database/exam/V1__exam_create_schema.sql

```diff
@@ -0,0 +1,582 @@
+
+--
+-- Table structure for table `field_test_item_group_event`
+--
+
+DROP TABLE IF EXISTS `field_test_item_group_event`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `field_test_item_group_event` (
+  `id` bigint(20) NOT NULL AUTO_INCREMENT,
+  `field_test_item_group_id` bigint(20) NOT NULL,
+  `deleted_at` datetime(3) DEFAULT NULL,
+  `position_administered` int(11) DEFAULT NULL,
+  `administered_at` datetime(3) DEFAULT NULL,
+  `created_at` timestamp(3) NOT NULL,
+  PRIMARY KEY (`id`),
+  KEY `field_test_item_group_id` (`field_test_item_group_id`),
+  CONSTRAINT `field_test_item_group_event_ibfk_1` FOREIGN KEY (`field_test_item_group_id`) REFERENCES `field_test_item_group` (`id`)
+) ENGINE=InnoDB;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Table structure for table `history`
+--
+
+DROP TABLE IF EXISTS `history`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `history` (
+  `id` varbinary(16) NOT NULL,
+  `client_name` varchar(100) NOT NULL,
+  `student_id` bigint(20) NOT NULL,
+  `subject` varchar(100) NOT NULL,
+  `initial_ability` float DEFAULT NULL,
+  `attempts` int(11) DEFAULT NULL,
+  `segment_id` varchar(200) DEFAULT NULL,
+  `changed_at` datetime(3) DEFAULT NULL,
+  `segment_key` varchar(250) DEFAULT NULL,
+  `exam_id` varbinary(16) DEFAULT NULL,
+  `tested_grade` varchar(50) DEFAULT NULL,
+  `login_ssid` varchar(50) DEFAULT NULL,
+  `item_group_string` text,
+  `initial_ability_delim` varchar(400) DEFAULT NULL,
+  PRIMARY KEY (`id`),
+  KEY `ix_historyexamid` (`exam_id`),
+  KEY `ix_historyloginssid` (`login_ssid`),
+  KEY `ix_studenthistory` (`student_id`,`client_name`,`subject`)
+) ENGINE=InnoDB;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Table structure for table `schema_version`
+--
+
+DROP TABLE IF EXISTS `schema_version`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `schema_version` (
+  `installed_rank` int(11) NOT NULL,
+  `version` varchar(50) DEFAULT NULL,
+  `description` varchar(200) NOT NULL,
+  `type` varchar(20) NOT NULL,
+  `script` varchar(1000) NOT NULL,
+  `checksum` int(11) DEFAULT NULL,
+  `installed_by` varchar(100) NOT NULL,
+  `installed_on` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  `execution_time` int(11) NOT NULL,
+  `success` tinyint(1) NOT NULL,
+  PRIMARY KEY (`installed_rank`),
+  KEY `schema_version_s_idx` (`success`)
+) ENGINE=InnoDB;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+INSERT INTO schema_version(installed_rank, version, description, type, script, checksum, installed_by, installed_on, execution_time, success)
+VALUES(1, 1496077806, 'exam field test item group tuning', 'SQL', 'V1__exam_create_schema.sql', -1515252824, 'V1__exam_create_schema', UTC_TIMESTAMP(), 201, 1);
```

Comment:
Wanted to call this out; I'm seeding Flyway's `schema_version` table to indicate the version of the schema.  The version (**1496077806**) is the correct version ('cause that's the most recent migration that was run), but I've set the `script` and `installed_by` to to indicate that the schema was created by this script.

It's a little confusing, I know - but I think setting the `version` to **1** will make it difficult for us to know where the `exam` schema's actually at.  If anyone has any other thoughts/ideas, I'm certainly willing to discuss them.